lib/cookbook-release/changelog.rb

```diff
@@ -120,9 +121,11 @@ def markdown_priority_nodes
       end
       if not_nodes_only_commits.any?
         result << "*#{NON_NODES_ONLY}*\n" << not_nodes_only_commits.map { |c| c.to_s_markdown(false) }.join("\n") << "\n"
+        result << "*#{NODES_ONLY}*\n" << cl.select(&:nodes_only?).map { |c| c.to_s_markdown(false) }.join("\n") << "\n"
+      else
+        result << "*#{FULL}*\n"
+        result << cl.map { |c| c.to_s_markdown(false) }.join("\n")
       end
-      result << "*#{FULL}*\n"
-      result << cl.map { |c| c.to_s_markdown(false) }.join("\n")
```

Comment:
Why keeping the `FULL` changelog when there is no "not-nodes-only-commits"?
IMHO we should have a coherent display:

```ruby
display_risky if risky_commits.any?
display_no_risky if risky_commits.empty? # I'm not sure about this one
display_nodes_only if nodes_only.any?
display_others if not_nodes_only_commits.any?
```

Reply:
good idea, done with different organization

Reply:
seems great